flask_monitoringdashboard/database/data_base_queries/mongo_db_objects.py

```diff
@@ -26,19 +27,59 @@ def _safe_mongo_call(*args, **kwargs):
     return _safe_mongo_call
 
 
+def safe_mongo_call_find(call, call_type):
+    """
+    In order to minimize the risk of failure and the impact on the app performance,
+    we need to allow the application to be able to read from the secondaries.
+    In such cases, we will have an eventual data consistence. Hence, we should re-try to read
+    """
+    def has_content(result):
```

Comment:
what type is `result` expected to be here? what is `len(list(result.clone())))` below ?

Reply:
result is a mongodb cursor, I clone it to return the data and the I need to convert to list to be able to know the number of elements. https://www.mongodb.com/docs/manual/reference/method/js-cursor/